docs/getting-started/5-output-rails/README.md

```diff
@@ -160,15 +161,67 @@ As we can see, the LLM did generate the message containing the word "idiot", how
 
 The following figure depicts the process:
 
-<div align="center">
-<img src="../../_static/puml/output_rails_fig_1.png" width="815">
-</div>
+```{image} ../../_static/puml/output_rails_fig_1.png
+```
+
+## Streaming Output
+
+By default, the output from the rail is synchronous.
+You can enable streaming to provide asynchronous responses and reduce the time to the first response.
+
+1. Modify the `rails` field in the `config.yml` file and add the `streaming` field to enable streaming:
+
+   ```{code-block} yaml
+   :emphasize-lines: 9,10,12
+
+   rails:
+     input:
+       flows:
+         - self check input
+
+     output:
+       flows:
+         - self check output
+       streaming:
+          window_size: 20
+
+   streaming: True
+   ```
+
+1. Call the `stream_async` method and handle the chunked response:
+
+   ```python
+   from nemoguardrails import RailsConfig, LLMRails
+
+   config = RailsConfig.from_path("./config")
+
+   rails = LLMRails(config)
+
+   messages = [{"role": "user", "content": "How many days of vacation does a 10-year employee receive?"}]
+
+   async for chunk in rails.stream_async(messages=messages):
+       print(f"CHUNK: {chunk}")
+   ```
+
+   *Partial Output*
+
+   ```output
+   CHUNK: Accor
+   CHUNK: din
+   CHUNK: g to
+   CHUNK:  the
+   CHUNK:  ABC Com
```

Comment:
running the example:

```output
CHUNK: According
CHUNK:  to
CHUNK:  the
CHUNK:  employee
CHUNK:  handbook,
CHUNK:  employees
CHUNK:  who
CHUNK:  have
CHUNK:  been
CHUNK:  with
CHUNK:  the
CHUNK:  ABC
CHUNK:  Company
CHUNK:  for
CHUNK:  10
CHUNK:  years
CHUNK:  or
CHUNK:  more
CHUNK:  are
CHUNK:  eligible
CHUNK:  for
CHUNK:  25
CHUNK:  days
CHUNK:  of
CHUNK:  paid
CHUNK:  vacation
CHUNK:  per
CHUNK:  year.
CHUNK:  Is
CHUNK:  there
CHUNK:  anything
CHUNK:  else
CHUNK:  I
CHUNK:  can
CHUNK:  assist
CHUNK:  you
CHUNK:  with?
```

Reply:
I can't rep your output, but I revised to show the first few whole words.  (Too bad for you. When I ran it, there was mention of a bonus in December. Tsk, tsk.)